dynamodb/migrate_dynamodb.py

```diff
@@ -23,8 +31,8 @@
     frame=dyf,
     connection_type="dynamodb",
     connection_options={
-        "dynamodb.output.tableName": "boards",
-        "dynamodb.throughput.write.percent": "1.0"
+        "dynamodb.output.tableName": table_name,
+        "dynamodb.throughput.write.percent": "0.5"
```

Comment:
why 0.5?

Reply:
after study, looks it's mean every executor, and we have 2 DPI each have 1 executor, so divide to half.